Tests/Controllers/ParkingSlotControllerTests.cs

```diff
@@ -550,10 +556,11 @@ public void GivenId_WhenGetDeleteIsCalled_ThenNotEmptyViewResultIsReturned()
             Assert.IsType<ViewResult>(result);
             var model = Assert.IsAssignableFrom<ParkingSlotDetailsVM>((result as ViewResult).Model);
 
-            Assert.Equal(Id, model.Id);
-            Assert.Equal(Number, model.Number);
-            Assert.Equal(ParkingZoneId, model.ParkingZoneId);
-            Assert.Equal(IsAvailableForBooking, model.IsAvailableForBooking);
+            Assert.Equal(expectedSlotVM.Id, model.Id);
+            Assert.Equal(expectedSlotVM.Number, model.Number);
+            Assert.Equal(_testZoneId, model.ParkingZoneId);
+            Assert.Equal(expectedSlotVM.IsAvailableForBooking, model.IsAvailableForBooking);
```

Comment:
Why are you asserting each property separately? You already have objects assertion on line 563